core/components/css-utilities/Flex/Flex.story.tsx

```diff
@@ -176,6 +176,179 @@ export const flex = () => {
       className: 'order-last',
       properties: 'order: 6 ;',
     },
+    /* New utility classes */
+    {
+      className: 'gap-0',
+      properties: 'gap: 0 ;',
+    },
+    {
+      className: 'gap-1',
+      properties: 'gap: var(--spacing-2-5) ; /* 1px */',
+    },
+    {
```

Comment:
Please create a separate table for each group of class to make it easier for better user discoverability like different tables for following group of classes:
- flex
- justify-content
- align-items
- align-content
- align-self
- order
- gap, row-gap, column-gap
- justify-items
- justify-self
- place-content
- place-items
- place-self